pkg/authorization/authorizer/browsersafe/authorizer_test.go

```diff
@@ -51,7 +51,7 @@ func TestBrowserSafeAuthorizer(t *testing.T) {
 		safeAuthorizer := NewBrowserSafeAuthorizer(delegateAuthorizer, "system:authenticated")
 
 		authorized, reason, err := safeAuthorizer.Authorize(tc.attributes)
-		if authorized == authorizer.DecisionAllow || len(reason) != 0 || err != nil {
+		if authorized == authorizer.DecisionAllow || err != nil {
```

Comment:
`if authorized != authorizer.DecisionNoOpinion || err != nil {`

The test should probably also check to make sure the `reason` string makes sense.

Reply:
if it were a generic test I would agree, but here seem really superfluous, what case would possibly generate a string we do not like ?